default.nix

```diff
@@ -86,12 +86,25 @@ stdenv.mkDerivation rec {
 
   prefixKey = "-prefix ";
 
-  enableParallelBuilding = true;
+  enableParallelBuilding = false;
```

Comment:
cc @ejgallego

Reply:
@Zimmi48 asked if this was necessary before.

Reply:
ok let's revert